Firebase/InAppMessaging/Flows/FIRIAMActivityLogger.m

```diff
@@ -106,7 +106,14 @@ - (instancetype)initWithMaxCountBeforeReduce:(NSInteger)maxBeforeReduce
                                              selector:@selector(appWillBecomeInactive)
                                                  name:UIApplicationWillResignActiveNotification
                                                object:nil];
-
+#if defined(__IPHONE_13_0) && __IPHONE_OS_VERSION_MAX_ALLOWED >= 130000
+    if (@available(iOS 13.0, *)) {
+      [[NSNotificationCenter defaultCenter] addObserver:self
```

Comment:
Are there situations where `appWillBecomeInactive` could be called twice, i.e. could `UIApplicationWillResignActiveNotification` and `UISceneWillDeactivateNotification` both be fired for the same app close?

Reply:
Same thing for becoming active.